config/crd/bases/core.oam.dev_componentschematics.yaml

```diff
@@ -326,7 +326,6 @@ spec:
                 required:
                 - image
                 - name
-                - resources
```

Comment:
this is auto generated, you could add `//+optional` annotation here https://github.com/oam-dev/oam-go-sdk/blob/master/apis/core.oam.dev/v1alpha1/component_schematic_types.go#L181

Reply:
And then re-generate these files by `make manifests`